test/controllers/payment.controller.spec.ts

```diff
@@ -129,5 +130,21 @@ describe("Payment controller", () => {
       expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(1);
       expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(CONFIRMATION_WITH_PARAMS_URL);
     });
+
+    test(`should redirect to ${PAYMENT_FAILED_PAGE} page, Payment failed somehow and feature flag active`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true);
```

Comment:
Would be good to get a comment at the end of this line to say what feature flag value is being mocked to 'true' here.

Reply:
Updated this.